test/extension/config/default/namespace.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: v1
```

Comment:
cf. namespace of the core provider
```yaml
apiVersion: v1
kind: Namespace
metadata:
  labels:
    control-plane: controller-manager
  name: system
```

Reply:
this is similar to https://github.com/kubernetes-sigs/cluster-api/pull/7288#discussion_r981587836 (clusterctl contract is not prescriptive on having this label)

Reply:
Should we have it anyway?

Reply:
Okay apparently that label is actually used:

```yaml
apiVersion: v1
kind: Namespace
metadata:
  labels:
    app.kubernetes.io/managed-by: tilt
    cluster.x-k8s.io/provider: cluster-api
    control-plane: controller-manager
    kubernetes.io/metadata.name: capi-system
  name: capi-system
```

Reply:
I would prefer having a follow-up (I can take it) were we sync up some of those things with our other providers. I know we don't need it, but it feels not ideal to me to diverge too much in the Runtime Extension

(we can keep the simple folder structure, I would just standardize some things like labels further)

Reply:
tracking this in https://github.com/kubernetes-sigs/cluster-api/issues/7322